dask/dataframe/partitionquantiles.py

```diff
@@ -451,6 +450,18 @@ def percentiles_summary(df, num_old, num_new, upsample, state):
                 )
                 .iloc[:, 0]
             )
+        else:
+            # Fall back to ndarray (Not supported for cuDF)
+            vals, _ = _percentile(array_safe(data, like=data.values), qs, interpolation)
+
+    # Convert to array if necessary (and possible)
+    if is_series_like(vals):
+        try:
+            vals = vals.values
+        except (ValueError, TypeError):
+            # cudf->cupy won't work if nulls are present,
+            # or if this is a string dtype
+            pass
```

Comment:
My sense is that this entire code block can/should be removed for the sake of simplicity. However, I'm currently  leaving the `.values` operation in place to keep the scope if this PR small.